pkg/services/pluginsintegration/clientmiddleware/logger_middleware.go

```diff
@@ -81,7 +81,11 @@ func (m *LoggerMiddleware) QueryData(ctx context.Context, req *backend.QueryData
 		ctxLogger := m.logger.FromContext(ctx)
 		for refID, dr := range resp.Responses {
 			if dr.Error != nil {
-				ctxLogger.Error("Partial data response error", "refID", refID, "error", dr.Error)
+				logParams := []any{"refID", refID, "error", dr.Error}
```

Comment:
Nit. Should we include the status field as well?

Reply:
Good idea. Even when the feature flag is off I suppose, like so:

```suggestion
				logParams := []any{"refID", refID, "status", requestStatusFromError(dr.Error), "error", dr.Error}
```

WDYT?

Reply:
No just `dr.Status`.

Reply:
Ah yes, makes sense. I will add that :+1:

Reply:
@marefr can you confirm the following

if we use

```suggestion
				logParams := []any{"refID", refID, "status", dr.Status, "error", dr.Error}
```

it produces:

```
Partial data response error              logger=plugin.instrumentation endpoint=queryData pluginId=grafana-datasourcehttpbackend-datasource dsName="Datasource Http Backend" dsUID=a3c4dc18-aac3-4250-978f-aae9ae281f4f uname=admin refID=A status="Bad Gateway" error="bad gateway request" statusSource=plugin
```

(`status="Bad Gateway"`)

because of: https://github.com/grafana/grafana-plugin-sdk-go/blob/04d7c57478a115779986260ecc519abc630ee0fe/backend/status.go#L82

Instead I suppose we want:

```suggestion
				logParams := []any{"refID", refID, "status", int(dr.Status), "error", dr.Error}
```

:point_down: 

```
Partial data response error              logger=plugin.instrumentation endpoint=queryData pluginId=grafana-datasourcehttpbackend-datasource dsName="Datasource Http Backend" dsUID=a3c4dc18-aac3-4250-978f-aae9ae281f4f uname=admin refID=A status=502 error="bad gateway request" statusSource=plugin
```

(`status=502`)

right?

Reply:
Yes, `int(dr.Status)` makes most sense :+1: